src/test/kotlin/blackjack/RoundTest.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+import java.lang.IllegalStateException
+
+class RoundTest {
+
+    @Test
+    fun `카드는 52장 뽑을 수 있다`() {
+        val cards = mutableListOf<Card>()
+        val round = Round()
+
+        repeat(52) {
+            cards.add(round.getCard())
+        }
+
+        cards.size shouldBe 52
+
+        // 52장 중복이 없는지 확인
+        cards.toSet().size shouldBe 52
```

Comment:
```suggestion
        cards.toSet().size shouldBe 52
```
set으로 변경했을 때, 크기가 52인지만 체크하면 되지 않을까요? 🤔

Reply:
논리적으로 봤을 때는 set으로 변환했을 때만 확인해도 문제 없는 것이 맞아요😅

리스트의 size를 체크해서 카드를 52장 뽑을 수 있다는 것을 보여주고, 그 뒤에 set으로 체크해봤을 때 중복도 없었다 라는 것을 보여주기 위해서 테스트 시나리오 상 의도적으로 넣은 코드라고 이해해주시면 감사하겠습니다ㅎㅎ

Reply:
좋습니다. 💯